slick/src/main/scala/slick/jdbc/meta/MColumn.scala

```diff
@@ -24,7 +24,7 @@ object MColumn {
           case _ => None
         }, r.<<, r.<<, r.skip.skip.<<, r.<<, DatabaseMeta.yesNoOpt(r),
         if(r.hasMoreColumns) MQName.optionalFrom(r) else None,
-        r.<<?,
+        r.<<?[Int],
```

Comment:
Changed this to use ```r.nextObjectOption``` instead of ```r.nextIntOption```.  All test seem to pass and I'm not sure if this will have any side effects.